spec/user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Alphonce', photo: 'https:/photo.com', bio: 'Teacher', posts_counter: 5) }
+
```

Comment:
Please add a test for `#recent_posts` in User model specs as any custom methods in the models should have corresponding unit tests.